modules/rostests/apitests/win32u/win32u_vista/CMakeLists.txt

```diff
@@ -9,3 +9,8 @@ add_library(win32u_vista MODULE
 
 set_module_type(win32u_vista module)
 add_dependencies(win32u_vista psdk)
+add_rostests_file(TARGET win32u_vista)
+
+if (STACK_PROTECTOR)
```

Comment:
`STACK_PROTECTOR` and `-fno-stack-protector` logically mismatch.

Reply:
That is probably to disable stack protector in this library

Reply:
Yes, I copy-pasted it from win32u_ros. @zefklop put it there to "Fix GCC -fstack-protector usage" in rev c8d07514c8e0bc235e180f6990f914ee86880f8c